.prettierignore

```diff
@@ -0,0 +1,9 @@
+.prettierignore
+package.json
```

Comment:
I can see from your various comments that you don't like Prettier. Your tone sounds like you're talking to me like I'm one of the authors of Prettier, I'm not. Ease up on the whole "docking points" talk.

This can be removed if the indent size for JSON in editorconfig matches the indent size in prettier.

Reply:
>I prefer an indent size of 4 for Javascript which is why I had package.json in my prettierignore.

your opinion forced it then?

Reply:
I copy-pasted my prettier ignore. Since the default indent has been decided at a
2, then this exclusion is not necessary in the ruleset.

Reply:
I've had experiences where `prettier` mis-formats `package.json` and pushes long commands to adhere to a line char limit. It wasn't ideal and did add `package*.json` to `.prettierignore`. Another reason to ignore: the IDE eslint/prettier validation service can be taxed with additional files that it does not need to formatted like a `package.json` or `package-lock.json`.